cmd/cloud-controller-manager/nodeipamcontroller.go

```diff
@@ -151,7 +182,8 @@ func startNodeIpamController(ccmConfig *cloudcontrollerconfig.CompletedConfig, n
 		serviceCIDR,
 		secondaryServiceCIDR,
 		nodeCIDRMaskSizes,
-		ipam.CIDRAllocatorType(ccmConfig.ComponentConfig.KubeCloudShared.CIDRAllocatorType),
+		allocatorType,
+		stackType,
```

Comment:
you already pass `clusterCIDRs`, `serviceCIDR` and `secondaryServiceCIDR` , there is no need to pass `stackType`, you can infer that from these parameters later in the controller

Reply:
Done